src/locales/es/interactives/sam/light-matter/HAS_sunlight_on_ground.json

```diff
@@ -10,6 +10,8 @@
   "fontScale": 0.8,
   "lang": "en-US",
   "i18nMetadata": "",
+    "lang": "es",
+  "i18nMetadata": "locales/metadata/interactives/sam/light-matter/HAS_sunlight_on_ground.json",
```

Comment:
You can see that in lines 11 and 12 we have the same properties already defined. It'd be good to have just one copy. I guess ones that are defined later would be used, but I'm not sure if it can be guaranteed and/or some browser won't complain about incorrect JSON format.